gdcdictionary/schemas/diagnosis.yaml

```diff
@@ -19624,6 +19695,34 @@ properties:
       - Unknown
       - Not Reported
 
+  uicc_staging_system_edition:
+    enum:
+      - 1st
+      - 2nd
+      - 3rd
+      - 4th
+      - 5th
+      - 6th
+      - 7th
+      - 8th
+      - Unknown
+      - Not Reported
+
+  weiss_assessment_findings:
+    type: array
+    items:
+      enum:
+        - Nuclear Grade III or IV
+        - Mitotic Rate > 5/50 HPF
+        - Atypical Mitotic Figures
+        - Cytoplasm presence <= to 25%
+        - Diffuse Architecture
+        - Necrosis
+        - Venous Invasion
+        - Sinusoid Invasion
+        - Invasion of Tumor Capsule
+        - Not Reported
+
   weiss_assessment_score:
```

Comment:
tissue_or_organ_of_origin new enums seem to be missing

Reply:
See my comment from 1/17 on diagnosis ticket (https://jira.opensciencedatacloud.org/browse/DICT-228), there were duplicate entries for upper/middle/lower third of esophagus caught by travis tests, i.e. values already exist in yaml file